### DISCUSSION TITLE
Option to auto-hide upvoted and downvoted posts, separately

### DISCUSSION THREAD

**Discussion by ghost** [Docile-Alligator/Infinity-For-Reddit#878] [Ideas]
Please grant us option to auto-hide upvoted posts, AND another to auto-hide downvoted posts.
For us that don't use mark read feature for reasons, and for us that doesn't want to look at downvoted posts ever again!

Thank you!

**Comment by Wladefant** [comment]
At the moment the mark as read feature works only locally and therefore you should not have any reasons to not use it.
